meta-lmp-base/recipes-support/libp11/files/0003-FIO-internal-ec_keygen-changes-required-for-TPM.patch

```diff
@@ -0,0 +1,46 @@
+From a4ca76ede26b5e2a54e914d3aa3bf10e40e66020 Mon Sep 17 00:00:00 2001
+From: Jorge Ramirez-Ortiz <jorge@foundries.io>
+Date: Sun, 16 Apr 2023 21:39:37 +0200
+Subject: [PATCH 3/4] [FIO internal] ec_keygen: changes required for TPM
+
```

Comment:
```
Upstream-Status: Pending
```

Reply:
I dont think this is ever going to be merged after all this time hence why I tagged it internal

Reply:
in this case `Upstream-Status: Inappropriate [lmp specific]` is better